components/raftstore/src/store/util.rs

```diff
@@ -934,6 +934,15 @@ pub fn check_conf_change(
             (ConfChangeType::RemoveNode, PeerRole::Voter) if kind != ConfChangeKind::Simple => {
                 return Err(box_err!("{:?}: can not remove voter directly", cp));
             }
+            (ConfChangeType::RemoveNode, PeerRole::Voter)
+                if kind == ConfChangeKind::Simple && current_voter.len() <= 2 =>
+            {
+                // Details are in Issue 10595 (https://github.com/tikv/tikv/issues/10595).
```

Comment:
Better give a brief explanation in comment.

Reply:
A brief explanation is added.